support-docs/examples.md

```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
     }
 }
 
-```
\ No newline at end of file
+```
+
+Inductor Example
+
+```json
+{
+    "componentID": {
+        "partType": "inductor",
+        "manufacturer": "ACME Components",
+        "componentName": "abc123",
+        "orderableMPN": "abc123,45",
+        "sourceDatasheetID": {
+            "publishedDate": "2021-05-23",
+            "datasheetURI": "www.acmecomponents.com/abc123datasheet"
+        },
+        "digitalDatasheetID": {
+            "publishedDate": "2022-03-21",
+            "guid": "3e4cd9de-657a-41ae-902e-beca95aff51d"
+        },
+        "status": "active",
+    },
+    "value": {
+        "siUnit": "henry",
+        "absoluteValue": 10,
+        "unitText": "uH",
+        "unitFactor": 1E-6,
+        "valueDefined": true
+    },
+    "ratedCurrent": {
+        "siUnit": "amp",
+        "absoluteValue": 4,
+        "unitText": "A",
+        "unitFactor": 1,
+        "valueDefined": true
+    },
+    "saturationCurrent": {
+        "siUnit": "amp",
+        "absoluteValue": 6,
+        "unitText": "A",
+        "unitFactor": 1,
+        "valueDefined": true
+    },
+    "saturationCurve": {
+        "title": "DC Bias Curve for abc123",
```

Comment:
Adding the "numberOfCurves" property may be good here, although this is not mandatory